### PR TITLE
Implement repair queue

### PR DESCRIPTION
**Pull request by morimoto-cybozu**
Signed-off-by: morimoto-cybozu <kenji_morimoto@cybozu.co.jp>

### CODE REVIEW

**Review by zeroalphat — APPROVED** [review]
LGTM

**Review by zeroalphat — DISMISSED** [review]
LGTM

File: op/status.go [hunk 1]
```diff
@@ -563,3 +565,102 @@ func containCommandOption(slice []string, optionName string) bool {
...
+	for _, entry := range entries {
+		// Update Nodename every time.
+		// Though the nodename of a machine in a Kubernetes cluster will not change,
+		// the machine can join/leave the cluster dynamically.
+		entry.FillNodename(cluster)
+	}
+
+	rqs.RepairCompleted = make(map[string]bool)
+	for _, entry := range entries {
+		if !(entry.Status == cke.RepairStatusQueued || entry.Status == cke.RepairStatusProcessing) {
```
Comment (masa213f) [inline op/status.go:598]: I thinsk this is easier to understand the meaning.

```suggestion
		if entry.HaveFinished() {
```

**Review by masa213f — CHANGES_REQUESTED** [review]
I'm sorry for the delayed review.

File: op/repair_drain_timeout.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		Target: c.entry.Address,
+	}
+}
+
+func repairDrainBackOff(ctx context.Context, inf cke.Infrastructure, entry *cke.RepairQueueEntry, err error) error {
+	log.Warn("failed to drain node for repair", map[string]interface{}{
+		"address":   entry.Address,
+		log.FnError: err,
+	})
+	entry.Status = cke.RepairStatusProcessing
```
[the remark this answers is not in the document]
Comment (masa213f) [reply op/repair_drain_timeout.go:64]: Thank you for the description. I understood.

When a node's repair waits for the drain in step 2, the CKE will execute the other node's repair.
And then, more entries than "max_concurrent_repairs" will be in the "processing" status.

**Review by masa213f — DISMISSED** [review]

**Review by YZ775 — APPROVED** [review]
LGTM

**Review by zeroalphat — APPROVED** [review]
LGTM

**Pull request merged**